.github/workflows/build-toolchain.yml

```diff
@@ -284,8 +299,10 @@ jobs:
             fi
           fi
 
-          echo windows_build_runner=${{ inputs.windows_default_runner || vars.WINDOWS_BUILD_RUNNER || 'windows-latest' }} >> ${GITHUB_OUTPUT}
-          echo windows_compilers_runner=${{ inputs.windows_compilers_runner || inputs.windows_default_runner || vars.COMPILERS_BUILD_RUNNER || vars.WINDOWS_BUILD_RUNNER || 'windows-latest' }} >> ${GITHUB_OUTPUT}
+          echo windows_x64_build_runner=${{ inputs.windows_x64_default_runner || vars.WINDOWS_BUILD_RUNNER || 'windows-latest' }} >> ${GITHUB_OUTPUT}
+          echo windows_x64_compilers_runner=${{ inputs.windows_x64_compilers_runner || inputs.windows_x64_default_runner || vars.COMPILERS_BUILD_RUNNER || vars.WINDOWS_BUILD_RUNNER || 'windows-latest' }} >> ${GITHUB_OUTPUT}
+          echo windows_arm64_build_runner=${{ inputs.windows_arm64_default_runner || vars.WINDOWS_BUILD_RUNNER || 'windows-latest' }} >> ${GITHUB_OUTPUT}
+          echo windows_arm64_compilers_runner=${{ inputs.windows_arm64_compilers_runner || inputs.windows_arm64_default_runner || vars.COMPILERS_BUILD_RUNNER || vars.WINDOWS_BUILD_RUNNER || 'windows-latest' }} >> ${GITHUB_OUTPUT}
```

Comment:
Should we rename the variables to be uniform? We cannot use the same name on both for the native build.

Reply:
`vars.WINDOWS_BUILD_RUNNER` and `vars.COMPILERS_BUILD_RUNNER` cannot be used simultaneously if you want native compilation. So, should these become `vars.WINDOWS_X64_DEFAULT_BUILD_AGENT` and `vars.WINDOWS_X64_COMPILERS_BUILD_AGENT`, augmented by `vars.WINDOWS_ARM64_{DEFAULT,COMPILERS}_BUILD_AGENT`.

Reply:
Done. Added those variables to `bcny/swift-build`. Do we need to worry about the variables on `compnerd/swift-build`?

Reply:
ok, based on offline chat, `compnerd/swift-build` does have these variables defined but they are less concern.